deps.edn

```diff
@@ -240,7 +240,7 @@
     djblue/portal                {:mvn/version "0.56.0"}                    ; ui for inspecting values
     hashp/hashp                  {:mvn/version "0.2.2"}                     ; debugging/spying utility
     io.github.camsaul/humane-are {:mvn/version "1.0.2"}
-    io.github.metabase/hawk      {:mvn/version "1.0.0"}
+    io.github.metabase/hawk      {:sha "8ecb6231ec9e8f51854fda03377c1be851b50d61"}
```

Comment:
This is a placeholder until someone finally reviews https://github.com/metabase/hawk/pull/31 (might take a long time)